com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -573,6 +573,8 @@ public static void main(String[] args) throws IOException, CompileExceptionError
         try (BufferedOutputStream os = new BufferedOutputStream(new FileOutputStream(outputPath))) {
 
             IShaderCompiler.CompileOptions compileOptions = new IShaderCompiler.CompileOptions();
+            compileOptions.forceIncludeShaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLSL_SM120);
```

Comment:
Force include glsl when compiling debug material.

Reply:
is it used by the editor?

Reply:
Used by `waf_render.py`. Editor doesn't invoke `ShaderProgramBuilder.main` (if I understand everything correctly)